docs/source/snakemake/debugging.md

```diff
@@ -35,7 +35,9 @@ your_workflow_name_jit_register_task(
 You can execute the script in your `latch develop` session like so:
 
 ```console
-$ python3 scripts/dry_run.py
+$ /usr/local/bin/python3 scripts/dry_run.py
```

Comment:
```suggestion
$ python3 scripts/dry_run.py
```

this is just not going to work for a lot of people

I think only Ubuntu `apt` installs Python here? in any case we should be promoting virtual env-based installs and not system-wide